### PR TITLE
Replace psutil usages in utils.stop_standing_subprocess.

### PR DESCRIPTION
**Pull request by eric100lin**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/787)

This PR is based on PR #786 .

### CODE REVIEW

File: mobly/utils.py [hunk 1]
```diff
@@ -257,6 +258,75 @@ def rand_ascii_str(length):
...
+    A list of pid of the descendant processes.
+  """
+  ret = []
+  stack = [starting_pid]
+
+  while stack:
+    pid = stack.pop()
+    try:
+      ps_results = subprocess.check_output([
+          'ps',
```
Comment (xpconanfan) [inline mobly/utils.py:279]: I see a couple of raw commands added.
do these commands work on windows?
and are they the same between mac and linux?

File: mobly/utils.py [hunk 2]
```diff
@@ -257,6 +258,75 @@ def rand_ascii_str(length):
 
 
 # Thead/Process related functions.
+def collect_process_tree(starting_pid):
```
Comment (xpconanfan) [inline mobly/utils.py:261]: make this module "private"?

**Review by xpconanfan — CHANGES_REQUESTED** [review]

File: mobly/utils.py (hunk 2 above)
Comment (eric100lin) [reply mobly/utils.py:261]: Done

File: mobly/utils.py (hunk 1 above)
Comment (eric100lin) [reply mobly/utils.py:279]: As the doc string, this function only available on Unix like system.
The command works on Unix like system as you can see all tests pass on Windows, Mac and Linux.

**Review by xpconanfan — APPROVED** [review]

**Pull request merged**